### ISSUE TITLE
if pdf attachments contain spaces, they cannot be opened

### ISSUE THREAD

**Issue by gitgerardo** [deltachat/deltachat-android#1345]
- Android version:10  (Omnirom)
- Device:Oneplus 6
- Delta Chat version:1.6.2
- Expected behavior:open pdf files 
- Actual behavior:if pdf attachments contain spaces, they cannot be opened
error message:" storage permission is needed to proceed "
- Steps to reproduce the problem: If I send a PDF attachment with spaces in the name, the attachment cannot be opened (error message:" storage permission is needed to proceed "). 
If I send the same attachment with a different name without a spaces, it opens.

**Comment by Hocuri** [comment]
I could not reproduce this on Android 9, Fairphone 3, DC 1.6.2, not even if I denied storage permission to DC.

Does this still happen if you allow storage permission for DC in the system settings?

**Comment by r10s** [comment]
may it be that the error comes from the app handling the pdf, not from Delta Chat.

@gitgerardo can you provide a screenshot?

**Comment by gitgerardo** [comment]
Yes , this still happens if I allow storage permission for DC in the system settings.

**Comment by gitgerardo** [comment]
> may it be that the error comes from the app handling the pdf, not from Delta Chat?
> 
> @gitgerardo can you provide a screenshot?

I tried to open  the two pdf attachments you see in the screenshot with different PDF viewer.It's not possible to open the second one .I have to save it before I can open it. After saving it has a very long name:
"utf-8''Risikoeinscha%CC%88tzungCoronaMusikSpahnRichterUpdate6.5.2020.%20pdf.pdf"

![Screenshot_Delta_Chat_20200511-165955](https://user-images.githubusercontent.com/33866993/81577985-8139e580-93aa-11ea-8b93-4deb1fb7f753.png)
![Screenshot_Xodo_Docs_20200511-170132](https://user-images.githubusercontent.com/33866993/81578078-a0d10e00-93aa-11ea-8053-86c979063c54.png)
